### PR TITLE
Remove unused v2 code from /website/common/script

### LINKED ISSUE #7993: Remove v2 code from /common/script
- Some ops/fns/... were only used in v2 and can be removed (client only code must stay for now)
- Some ops return different data if `req.v2 === true`, that code can be removed

### PR DESCRIPTION
**Pull request by khipkin**
Fixes #7993
### Changes

Removes unused code from `/website/common/script/fns` and `/website/common/script/ops`.

@paglias

### CODE REVIEW

**Review by crookedneighbor — COMMENTED** [review]
From a quick glance, it looks reasonable to me. I'll try to pull it down and double check it in the next few days.

File: website/common/script/ops/getTags.js [hunk 1]
```diff
@@ -1,5 +0,0 @@
-// TODO used only in client, move there?
```
Comment (paglias) [inline website/common/script/ops/getTags.js:1]: Wasn't this used in the client?

File: website/common/script/ops/getTag.js [hunk 2]
```diff
@@ -1,18 +0,0 @@
-import _ from 'lodash';
-import i18n from '../i18n';
-import { NotFound } from '../libs/errors';
-
-// TODO used only in client, move there?
```
Comment (paglias) [inline website/common/script/ops/getTag.js:5]: Wasn't this used in the client?

File: website/common/script/ops/update.js [hunk 3]
```diff
@@ -1,11 +0,0 @@
-import _ from 'lodash';
-
-// TODO used only in client, move there?
```
Comment (paglias) [inline website/common/script/ops/update.js:3]: Wasn't this used in the client?

**Review by paglias — APPROVED** [review]

File: website/common/script/ops/getTags.js (hunk 1 above)
Comment (khipkin) [reply website/common/script/ops/getTags.js:1]: I couldn't find any place where it was used.

File: website/common/script/ops/getTag.js (hunk 2 above)
Comment (khipkin) [reply website/common/script/ops/getTag.js:5]: I couldn't find any place where it was used.

File: website/common/script/ops/update.js (hunk 3 above)
Comment (khipkin) [reply website/common/script/ops/update.js:3]: I couldn't find any place where it was used.

File: website/common/script/ops/getTag.js (hunk 2 above)
Comment (crookedneighbor) [reply website/common/script/ops/getTag.js:5]: I can't find any place it is used either.

File: website/common/script/ops/getTags.js (hunk 1 above)
Comment (crookedneighbor) [reply website/common/script/ops/getTags.js:1]: Confirmed. Not used as far as I can tell.

File: website/common/script/ops/update.js (hunk 3 above)
Comment (crookedneighbor) [reply website/common/script/ops/update.js:3]: Confirmed. Not used.

**Comment by crookedneighbor** [comment]
Well done. 

Tier awarded! You can read more about [contributor tiers and rewards here](http://habitrpg.wikia.com/wiki/Contributor_Rewards).

Note that as your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required. There's some information about that at Contributor Rewards, but feel free to ask here if you have questions!

**Comment by khipkin** [comment]
Eeeeee thanks!

**Pull request merged**